packages/tupaia-web/src/features/MapOverlays/PolygonLayer.tsx

```diff
@@ -23,22 +24,30 @@ const ChildEntities = ({ entities }: { entities: EntityResponse['children'] }) =
   );
 };
 
-const SiblingEntities = ({ entity }: { entity: EntityResponse }) => {
+const SiblingEntities = ({
+  parentEntityCode,
+  activeEntityCode,
+}: {
+  parentEntityCode: Entity['code'];
+  activeEntityCode: Entity['code'];
```

Comment:
I believe we have a helper type of `EntityCode` which you could also use here if you wanted, but it's not imperative.

I also tend to like to go by the rule of 'if a component has more than 1 prop, create an interface for it' so if you wanted to, you could add one here.